README.md

```diff
@@ -1,5 +1,8 @@
 ## Wise words ##
 
+
+![WHATEVS](/assets/crows.jpeg "crows")
+
```

Comment:
does it need cropping more?

Reply:
Um, no

Reply:
Let me rephrase that. Please crop the image more tightly on the crows.

Reply:
how is it now?

Reply:
better, thanks